packages/@okta/vuepress-site/docs/guides/step-up-authentication/main/index.md

```diff
@@ -33,7 +33,7 @@ Okta has created predefined `acr_values` that are described in the [Predefined p
 
 > **Note:** You can specify a `max_age` parameter value to require an elapsed time frame. Additionally, if you want to ignore the existing session and reauthenticate the user each time, pass `max_age=0` in the request. For Classic Engine, pass `max_age=1`. See the [Request parameters table](/docs/reference/api/oidc/#request-parameters) for the `/authorize` endpoint for more information on `max_age`.
 
-Okta's [redirect deployment model](/docs/concepts/redirect-vs-embedded/#redirect-authentication) supports the use of the `acr_values` parameter. The parameter works with any OpenID Connect application, such as web, native, or SPA, and it’s supported by both the [Okta Org Authorization Server and custom authorization servers](/docs/concepts/auth-servers/).
+Okta's [redirect and embedded](/docs/concepts/redirect-vs-embedded/#redirect-authentication) deployment models support the use of the `acr_values` parameter. The parameter works with any OpenID Connect application, such as web, native, or SPA, and it’s supported by both the [Okta Org Authorization Server and custom authorization servers](/docs/concepts/auth-servers/).
```

Comment:
looks like the link points to `#redirect-authentication` section, should point to the page in general since we are mentioning both models now?